app/src/main/java/org/jellyfin/mobile/webapp/RemoteVolumeProvider.kt

```diff
@@ -0,0 +1,25 @@
+package org.jellyfin.mobile.webapp
+
+import android.media.AudioManager
+import android.media.VolumeProvider
+import org.jellyfin.mobile.utils.Constants
+
+class RemoteVolumeProvider(private val webappFunctionChannel: WebappFunctionChannel) : VolumeProvider(VOLUME_CONTROL_ABSOLUTE, 100, 0) {
+    override fun onAdjustVolume(direction: Int) {
+        when (direction) {
+            AudioManager.ADJUST_RAISE -> {
+                webappFunctionChannel.triggerInputManagerAction(Constants.INPUT_MANAGER_COMMAND_VOL_UP)
+                currentVolume += 2 // TODO: have web notify app with new volume instead
+            }
+            AudioManager.ADJUST_LOWER -> {
+                webappFunctionChannel.triggerInputManagerAction(Constants.INPUT_MANAGER_COMMAND_VOL_DOWN)
+                currentVolume -= 2 // TODO: have web notify app with new volume instead
+            }
+        }
+    }
+
+    override fun onSetVolumeTo(volume: Int) {
+        webappFunctionChannel.callPlaybackManager("sendCommand({Name:'SetVolume', Arguments:{Volume:$volume}})")
```

Comment:
```suggestion
        webappFunctionChannel.callPlaybackManager("sendCommand({ Name: 'SetVolume', Arguments:{ Volume: $volume }})")
```

Would it be worth adding additional methods to the function channel to generate these commands so we don't have hardcoded strings like this one everywhere?

Reply:
Yeah, I think that makes sense..